docs/index.md

```diff
@@ -1,52 +1,71 @@
 # Chain-Aware ROS Evaluation Tool (CARET)
 
-コールバックチェーンを考慮した ROS 2 アプリケーションの性能評価ツールです。  
-フックでのトレースポイント追加により、メッセージのトラッキングをベースとした性能の評価が可能です。
+CARET is one of performance analysis tool dedicated with ROS 2 applications. It is able to measure not only callback latency and communication latency, but also path latency, in other words, chain of node or callback. As additional tracepoints are introduced by function hook, tracing resolution is improved.
 
-主な特徴
+Features and capabilities are shown below.
 
-- 以下の対象について、評価および測定が可能
+Features:
 
-  - コールバックの実行時間
-  - 通信レイテンシ
-    - メッセージのロスト
-  - ノードレイテンシ
-  - End-to-End レイテンシ
-  - エグゼキュータのスケジューリング
+- Low overhead with LTTng-based tracepoints for sampling events in ROS/DDS layer
+- Flexible tracepoints added by function hooking with LD_PRELOAD
+- Python-based API for flexible data analysis and visualization
+- Application-layer events tracing by cooperation with TILDE, runtime message tracer
 
-- LTTng (ros2_tracing) のトレースポイントによる低オーバーヘッド
+Capabilities:
 
-## 性能評価の流れ
+- Performance measurement from several aspects
+  - Callback latency, frequency, and jitter
+  - Topic communication latency, frequency, and jitter
+  - Node latency
+  - Path latency
+    - End-to-end latency of software if path from input to output is selected
+- Visualization of scheduling for callback execution
+- Filtering function to ignore specific nodes and topics
+- Search of target paths to trace
+- Trace of application events like consumption of buffered topic message
+  - `/tf` (planned for v0.3.x release)
+  - `message_filters` (supported by TILDE)
+  - `image_transport` (supported by TILDE)
+
+## Tracing flow with CARET
 
 ![measurement_flow](./imgs/measurement_flow.svg)
 
-CARET は ros2_tracing により追加されたトレースポイントに加え、  
-ROS や DDS の処理に対してトレースポイントを追加することで、様々な処理かかる時間の測定を可能にしています。
+CARET gives you capability of tracing your application with introducing new tracepoints to ROS and DDS layer while it utilized original tracepoints for [`ros2_tracing`](https://gitlab.com/ros-tracing/ros2_tracing).
 
-動的ライブラリの処理をフックするためのトレース用ライブラリに加え、  
-フォークしてトレースポイントを追加した ROS を必要とします。  
-現在はソースコードでの提供のみになっているため、これらパッケージのビルドが必要です。
+CARET is served as only source code, but not as `apt` package, so far.  
+CARET hooks dedicated functions to those defined in dynamic library in order to add tracepoints.  
+The fork of rclcpp which has CARET-dedicated tracepoints is delivered.
+You have to build CARET and your application if you want to use.
 
-測定を行うとトレース結果が得られ、この結果を元に、解析を行います。  
-解析前には、レイテンシの定義や測定対象のパスの定義を記述したアーキテクチャファイルが必要になります。
+If you run your application with CARET, you will get a set of trace data. If you analyze the data set, you have to create an architecture file in which node latency and target path are defined.
```

Comment:
Personally, I find the following text easier to read.
> you have to create an architecture file which defines node latency and target path.

Reply:
How about updated text? I think that "user has to define node latency and target path", but a file does not define.

Reply:
make sense.